dev-cpp/prometheus-cpp/prometheus-cpp-0.9.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Prometheus Client Library for Modern C++"
+HOMEPAGE="https://github.com/jupp0r/prometheus-cpp"
+SRC_URI="https://github.com/jupp0r/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+pull +push test zlib"
```

Comment:
These aren't very descriptive... What do they actually do? Should they be enabled unconditionally instead?

Reply:
`pull` will build the `libprometheus-cpp-pull.so.0.9.0` library, while `push` will build the `libprometheus-cpp-push.so.0.9.0` library;  these respectively enable clients to pull and push data from/to a prometheus server. These two libraries seem to be relatively small (36K and 76K respectively) so I don't think it'll be that bad to just build them unconditionally. I'll remove these flags then.